test/TensorFlowRuntime/tensor.swift

```diff
@@ -778,6 +778,25 @@ func testShapeGetter4() {
 }
 TensorTests.testAllBackends("ShapeGetter4", testShapeGetter4)
 
+@inline(never)
+func testReplacing1() {
+  let original = Tensor<Float>([[1.0, 0.0], [3.0, 0.0], [2.0, 3.0], [1.0, 0.0]])
+  let modified = original.replacing(with: Tensor<Float>(-1).broadcast(like: original), where: original.<3)
+  let expected = Tensor<Float>([[-1.0, -1.0], [3.0, -1.0], [-1.0, 3.0], [-1.0, -1.0]])
+  expectEqual(expected, modified)
+}
+TensorTests.testAllBackends("Replacing1", testReplacing1)
+
+@inline(never)
+func testReplacing2() {
+  // Implicit broadcast
```

Comment:
This comment can be included in the test name, as demonstrated in the comment above.